FinniversKit/Sources/Assets/ImageAsset.swift

```diff
@@ -56,6 +56,7 @@ enum ImageAsset: String {
     case checkCircleFilled
     case checkCircleFilledMini
     case checkmarkBig
+    case checmark
```

Comment:
Just a spelling-error: should be checkmark ✔

Reply:
Good catch😄 thanks!